### PR TITLE
Ping users joining channel if he have a pending %tell

### PR DESCRIPTION
**Pull request by zehortigoza**
My first time playing with go so fell free to suggest any improvements.

### CODE REVIEW

File: tell.go [hunk 1]
```diff
@@ -87,6 +94,20 @@ type tellRecord struct {
 	read              bool
 }
 
+func tellHaveMsg(db *sql.DB, ch, dst string) bool {
+	rows, err := db.Query("SELECT id, ch, src, dst, msg, dt, read FROM tell WHERE ch = ? AND read = 0 AND dst = ? ORDER BY dt", ch, dst)
```
Comment (edersondisouza) [inline tell.go]: If all you want is a `count`, i guess you don't need to select all those fields.

Comment (zehortigoza) [inline tell.go]: yes, fixed.
I was thinking in use "SELECT COUNT(*)..." but this way is more simple.

Comment (edersondisouza) [inline tell.go]: Remember that if there's a lot of registers on db, `select count()` would be less expensive =D
Or, you could add `LIMIT 1` on the query.

Comment (zehortigoza) [inline tell.go]: Well remembered, LIMIT 1
I lost almost all my knowledge in SQL and I think it is a good thing xP

**Comment by edersondisouza** [comment]
On 24-02-2017 15:48, José Roberto de Souza wrote:
> *@zehortigoza* commented on this pull request.
>
> --------------------------------------------------------------------------------------------
>
> In tell.go <https://github.com/cmarcelo/ircbot/pull/1#discussion_r103056855>:
>
>> @@ -87,6 +94,20 @@ type tellRecord struct {
>  	read              bool
>  }
>
> +func tellHaveMsg(db *sql.DB, ch, dst string) bool {
> +	rows, err := db.Query("SELECT id, ch, src, dst, msg, dt, read FROM tell WHERE ch = ? AND read = 0 AND dst = ? ORDER BY dt", ch, dst)
>
> Well remembered, LIMIT 1
> I lost almost all my knowledge in SQL and I think it is a good thing xP

I just remembered that there was something like that, google ftw =D

**Comment by zehortigoza** [comment]
I just wanted to have friends that review and integrate my code.

**Pull request merged**